### ISSUE TITLE
COPY FROM silently fails if timestamps cannot be converted

### ISSUE THREAD

**Issue by SneakyMax** [crate/crate#6196]
**CrateDB version**:  2.0.7 (however I didn't see any changes to COPY FROM in the newer changelogs)

**JVM version**:
java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)

**OS version / environment description**:
Ubuntu 16.04.2 LTS

**Problem description**:
When importing a bulk dataset using COPY FROM, I had created a schema beforehand with timestamp entries. It turned out that the data I was importing did not have dates in ISO-8601 format. The COPY FROM command just ends with `COPY OK (2.629 sec)` and no other data on why nothing was imported from the file (even though Crate was obviously reading the data).

The crate logs are mostly empty and don't say anything useful.

It would be great to have some indication that X rows couldn't be imported because of a data type mismatch, or something.

**Steps to reproduce**:

1. `create table test ( t timestamp )`
2. Create a json file like `{ t: '2017-07-24 00:15:26.649303' }`
3. Try to import that file into crate using COPY FROM
4. See **COPY OK** but nothing has been imported.

**Comment by joemoe** [comment]
hi @SneakyMax, we are returning the number of affected rows, which is the only possibility we have within the sql standard. what we could do and think of is logging the stats more explicitly. 

```
copy dates from '/Users/johannes/sandbox/tests/copy_from/dates.txt';
COPY OK, 6 rows affected  (0.022 sec)
```

As the `copy from` is intended for fast, highly optimized data transfer, there are almost no checks done. To achieve what you want to you need to compare the number of lines of the files and the number of affected rows.

**Comment by SneakyMax** [comment]
Yeah, I understand there's not much room here. I spent a whole day struggling to figure out why my data wasn't importing though. Only once I created a separate table without a schema did the import work, then I found out about the timestamp mismatch. The `COPY OK` was just super misleading :/

**Comment by joemoe** [comment]
we will improve the documentation and think about being more verbose in the logs.

**Comment by mfussenegger** [comment]
We've added a  `RETURN SUMMARY` clause to `COPY FROM` which will be available in the upcoming `3.1` which will cause `COPY FROM` to return more detailed information about what was going on. That should help debug such issues so I'm closing this for now.
